luigi/notifications.py

```diff
@@ -373,6 +376,11 @@ def format_task_error(headline, task, command, formatted_exception=None):
     :return: message body
     """
 
+    if formatted_exception:
+        if len(formatted_exception) > email().traceback_max_length:
+                truncated_exception = formatted_exception[:email().traceback_max_length]
```

Comment:
Flake8 complains that it is `./luigi/notifications.py:381:17: E117 over-indented`

Reply:
Ah, I will fix this